docs/_bugs/BUG-012-missing-market-status-enum.md

```
@@ -0,0 +1,67 @@
+
+The Kalshi API returns markets with `status: "initialized"` for new markets that haven't opened yet. Our enum only included:
+- `active`
+- `closed`
+- `determined`
+- `finalized`
+
+---
+
+## Error Message
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)